language/German/strings.po

```diff
@@ -2500,11 +2550,11 @@ msgstr "Beide ausblenden"
 
 #: src/frontend/mame/ui/custui.cpp:143
 msgid "UI Language"
-msgstr ""
+msgstr "UI Sprache"
```

Comment:
"UI" was translated to "Benutzeroberfläche" before, now it's just "UI"?

Reply:
Fixed